doc-style/action.yml

```diff
@@ -51,9 +51,7 @@ inputs:
   vale-version:
     description: >
       Version number for Vale.
-    # TODO: use the latest stable version from Vale, see issue
-    # https://github.com/ansys/actions/issues/350
-    default: '2.29.6'
+    default: '3.3.0'
```

Comment:
@dipinknair, should we update this to `3.4.1`? This is so far the latest version in Vale. If so, feel free to update and merge this. We are likely to release v6 next week.